src/common/hooks/useResolveDesign.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useDesignsQuery } from '../queries/designs';
+
+export const useResolveDesign = () => {
```

Comment:
Please follow the conventions of the project and use `function $name`.

Reply:
Also, can you explain why we use these fns? Looks to me they're simple .find, why need for _another_ method?

Reply:
@beganovich Done! Actually, those functions are created to resolve entities by IDs or some other specific values, just like all the others, for example:

```
export function useResolveLanguage() {
  const langauges = useLanguages();

  return (id: string | number) => {
    return langauges.find(
      (language) => language.id.toString() === id.toString()
    );
  };
}
```

Yeah, they look pretty simple, but I created hooks from them because it doesn't seem to me like a part of the `group_settings` logic. IMO, it should be separated from the logic of group_settings. Additionally, like other resolving hooks we've created, these can be used across other functionalities instead of duplicating the logic under every other functionality where it is needed. These functions are actually used within the `useResolvePropertyValue` hook, which you can find here: (https://github.com/Civolilah/ui/blob/feature/726-group-settings/src/pages/settings/group-settings/common/hooks/useResolvePropertyValue.ts)

This hook is used to resolve values from IDs and other specific values into values that should be shown to the user, just like in the Flutter app from the given `group_settings` object.

I hope you agree with me on using this approach to resolve values and separating it from the logic of the group_settings logic. Let me know your thoughts.